build/scraper.mjs

```diff
@@ -134,9 +135,7 @@ class Scraper {
    * @property {string} textureLocation location of the image for the corresponding item
    */
   /**
-   * @typedef {Object} ImageManifest
-   * @property {ImageManifestItem[]} Manifest
-   */
+   * */
```

Comment:
I'm not even sure what happened here. It looks like it was changed by the formatter but it removed that whole comment... I'm seeing why now and it says because it wasn't declared so I'm deferring to you if you want me to clean up the empty comment or add it back

Reply:
I'd prefer it stick around. typedefs exist to create type definitions that are for the internal mechanism of the library in this case for a bunch of stuff that exist for their own sake. i'm not sure why vs code is telling you it's not defined... this is how typedefs work in jsdoc, lol